src/client/containers/LandingPage/LandingPage.Container.js

```diff
@@ -1,11 +1,46 @@
 import React from 'react';
 import './LandingPage.Style.css';
+import HeroImage from '../../components/HeroImage/HeroImage.component';
+import Carousel from '../../components/Carousel/Carousel.component';
+import ProductView from '../../components/ProductView/ProductView.component';
 
 const LandingPageContainer = () => {
+  const [products, setProducts] = React.useState();
+  const [productsImages, setProductsImages] = React.useState();
+  const [categories, setCategories] = React.useState();
+  const [isLoading, setIsLoading] = React.useState(false);
+
+  React.useEffect(() => {
+    fetch('/api/products')
+      .then((res) => res.json())
+      .then((data) => {
+        setProducts(data);
+        setProductsImages(data.map((product) => product.picture));
+        setIsLoading(true);
+      });
+    fetch('/api/categories')
+      .then((res) => res.json())
+      .then((data) => setCategories(data));
```

Comment:
You also need categories isLoaded before you pass them to ProductView.

Reply:
Are you sure that? Because category page is not giving any problem without loading. Category is working when we use category in sort component. So category is being fetched already

Reply:
It is just a coincidence that categories are already fetched. But  you cannot predict that, if there will be error during categories fetch, or if something goes wrong (no table for example) the whole Component ProductView will through error. It is just a thing that can spare us from having bugs later.